sdcm/cluster.py

```diff
@@ -3957,6 +3957,9 @@ def __init__(self, *args, **kwargs):
         self._node_cycle = None
         self.params = kwargs.get('params', {})
         self.parallel_node_operations = self.params.get("parallel_node_operations") or False
+        if any([self.params.get('client_encrypt'), self.params.get('server_encrypt'),
+                self.params.get('internode_encryption')]):
+            create_ca(self.test_config.tester_obj().localhost)
```

Comment:
I think this can be move to BaseCluster, or even back to tester, if it's guarded like this 

in general we should avoid doing too much actions inside `__init__`, when doing unit tests it tend to complex things

Reply:
+1
it uses `tester_obj` so possibly `Tester` is better place

Reply:
Reverted this back